resources/dicts/events/new_cat/general.json

```diff
@@ -411,6 +426,10 @@
         "season": ["any"],
         "weight": 20,
         "event_text": "m_c finds a rogue named n_c_pre, who joins the Clan. The rogue decides to take on a more Clan-like name, and is now called n_c.",
```

Comment:
n_c abbrs need to follow the abbrs that patrols have been using. that is, the `n_c:{index}` system.  n_c_pre can be converted to `n_c_pre:{index]`